ginga/util/io_rgb.py

```diff
@@ -47,7 +47,7 @@
 #have_pil = False
 #have_cms = False
 #have_exif = False
-
+#have_opencv = False
```

Comment:
I think this should fix the PEP 8 failure?
```suggestion
#have_opencv = False

```

Reply:
Just pushed a fix for that, let's see if Travis is happy...